coralproject/talk
# Configure override css file

## What does this PR do?

we want to allow publishers to override styles in the Embed Stream without having to re-compile the app. This PR adds a setting to include an extra stylesheet if defined. The stylesheet will go after all the other styles, and should allow precedence.

## How do I test this PR?

- find a hosted stylesheet somewhere (maybe a CDN for a framework or something
- go to the Configure page. all the way at the bottom, enter your custom css url
- load the embed stream. it should have some styles overridden from your stylesheet.
